### ISSUE TITLE
Jitsi SIP config

### ISSUE THREAD

**Issue by tomkel** [spantaleev/matrix-docker-ansible-deploy#488]
I'm new to telephony, so I'm sorry for the vague question, but how would one go about setting up a SIP bridge using this playbook? It seems very complicated. Maybe we could try to collect some knowledge here and turn it into a file in `docs/`.

Here is the first link, which has to do with enabling PIN entry when calling in.
https://community.jitsi.org/t/how-to-enable-pin-for-sip-dial-in/18307
https://community.jitsi.org/t/jitsi-dev-ivr-dial-in-setup/13598

**Comment by spantaleev** [comment]
Our setup is inspired and very similar to the [docker-jitsi-meet](https://github.com/jitsi/docker-jitsi-meet) setup, which is [docker-compose](https://docs.docker.com/compose/)-based. It's worth looking at how they do it there.

Looks like one needs to add and configure the `jigasi` component for SIP audio (here's the `docker-compose` [file](https://github.com/jitsi/docker-jitsi-meet/blob/master/jigasi.yml) for that).

The playbook does not currently install `jigasi`, because it's something additional that I (and others) haven't found a need for. Installing and configuring `jigasi` is certainly something which can be added to our `matrix-jitsi` Ansible role though, as an optional component.

**Comment by tomkel** [comment]
Yes, jigasi seems to be one component, perhaps the component. According to [this](https://community.jitsi.org/t/how-to-enable-pin-for-sip-dial-in/18307/2) it appears that there is a "cloud API" that jitsi has created in order to display the phone numbers in the jitsi UI.
